services/core-web/src/components/modalContent/EditSitePropertiesModal.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SitePropertiesForm from "@/components/Forms/permits/SitePropertiesForm";
+
+const propTypes = {
+  onSubmit: PropTypes.func.isRequired,
```

Comment:
These prop types aren't being referenced anywhere in the file and can be removed, or, should we more clearly define them and pass them into the form individually?

Reply:
updated to pass the props individually